packages/blade/src/components/DatePicker/types.ts

```diff
@@ -213,7 +213,15 @@ type DatePickerInputProps = DatePickerCommonInputProps &
 type DatePickerFilterChipProps = DatePickerInputProps;
 
 type FilterChipDatePickerProps = Omit<DatePickerProps<'single' | 'range'>, 'label'> & {
+  /**
+   * Sets the label for the input element.
+   */
   label: string;
+  /**
+   *
+   * Callback which fires when clear button is clicked
+   */
+  onClearButtonClick?: () => void;
```

Comment:
I don't see any clear button 🤔 what is clear button on datepicker?

Reply:
this is a part of `FilterChipDatePickerProps` in case of  FilterChipDatePicker we have a clear button 
```
type FilterChipDatePickerProps = Omit<DatePickerProps<'single' | 'range'>, 'label'> & {
  /**
   * Sets the label for the input element.
   */
  label: string;
  /**
   *
   * Callback which fires when clear button is clicked
   */
  onClearButtonClick?: () => void;
};
```